client/a8c-for-agencies/sections/sites/sites-dataviews/style.scss

```diff
@@ -0,0 +1,215 @@
+		display: none;
+	}
+
+	thead {
+		th.sites-dataviews__site {
+			background: var(--studio-white);
+			td {
+				padding: 16px 0;
+				border-bottom: 1px solid var(--studio-gray-5);
+			}
+		}
+	}
+
+	tr.dataviews-view-table__row {
+		background: var(--studio-white);
+
+		.components-checkbox-control__input {
+			opacity: 0;
+		}
+		.components-checkbox-control__input:checked,
+		.components-checkbox-control__input:indeterminate, {
+			opacity: 1;
+		}
+
+		.dataviews-view-table-selection-checkbox {
+			padding-left: 12px;
+			&.is-selected {
+				.components-checkbox-control__input {
+					opacity: 1;
+				}
+			}
+		}
+
+		&:hover {
+			background: var(--studio-gray-0);
+
+			.site-set-favorite__favorite-icon,
+			.components-checkbox-control__input {
+				opacity: 1;
+			}
+		}
+
+		th {
+			padding: 16px 4px;
+			border-bottom: 1px solid var(--studio-gray-5);
+			font-size: rem(13px);
+			font-weight: 400;
+		}
+
+		td {
+			padding: 16px 4px;
+			border-bottom: 1px solid var(--studio-gray-5);
+			vertical-align: middle;
+		}
+
+		td:has(.sites-dataviews__site) {
+			padding: 8px 16px 8px 4px;
+			width: 20%;
+		}
+
+		.site-sort__clickable {
+			cursor: pointer;
+			padding-left: 1rem;
+
+		}
+
+		.dataviews-view-table__checkbox-column,
+		.components-checkbox-control__input {
+			label.components-checkbox-control__label {
+				display: none;
+			}
+			&[type="checkbox"] {
+				border-color: var(--studio-gray-5);
+			}
+		}
+	}
+
+	ul.dataviews-view-list {
+		li:hover {
+			background: var(--studio-gray-0);
+		}
+		.is-selected {
+			background-color: var(--studio-jetpack-green-0);
+		}
+	}
+}
+
+.components-search-control input[type="search"].components-search-control__input {
+	background: var(--studio-white);
+	border: 1px solid var(--studio-gray-5);
+}
+
+.dataviews-filters__view-actions {
+	margin-bottom: 8px;
+}
+
+.sites-dataviews__site {
+	display: flex;
+	flex-direction: row;
+
+	.button {
+		padding: 0;
+	}
+}
+
+.sites-dataviews__site-name {
+	display: inline-block;
+	text-align: left;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	white-space: nowrap;
+	width: 180px;
+	font-weight: 500;
+	font-size: rem(14px);
+}
+
+.sites-dataviews__site-url {
+	text-overflow: ellipsis;
+	overflow: hidden;
+	white-space: nowrap;
+	font-size: rem(12px);
+	color: var(--studio-gray-60);
+	font-weight: 400;
+}
+
+.preview-hidden {
+	@media (min-width: 2040px) {
+		.sites-dataviews__site-name {
+			width: 250px;
+		}
+	}
+}
+
+.sites-dataviews__actions {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: center;
+	flex-wrap: nowrap;
+
+	@media (min-width: 1080px) {
+		.site-actions__actions-large-screen {
+			float: none;
+			margin-inline-end: 20px;
+		}
+	}
+
+	.site-preview__open {
+		.gridicon.gridicons-chevron-right {
+			width: 18px;
+			height: 18px;
+		}
+	}
+}
+
+.dataviews-pagination {
+	margin-bottom: 2rem;
+}
+
+
+.dataviews-wrapper {
+	.dataviews-no-results,
+	.dataviews-loading {
+		padding-top: 1rem;
```

Comment:
[Non-blocker] It is better to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) everywhere on this file since we can support RTL locales in Calypso.